store/sqlstore/tokens_store.go

```diff
@@ -74,10 +74,10 @@ func (s SqlTokenStore) GetByToken(tokenString string) (*model.Token, error) {
 	return &token, nil
 }
 
-func (s SqlTokenStore) Cleanup() {
+func (s SqlTokenStore) Cleanup(expireTime int64) {
 	mlog.Debug("Cleaning up token store.")
```

Comment:
Let's move this debug line to the `doTokenCleanup` function before calling this method.

Reply:
Done.